tests/e2e/specs/utils/confluent.ts

```diff
@@ -17,11 +17,4 @@ export async function openConfluentExtension(page: Page): Promise<void> {
     state: "visible",
     timeout: 30_000,
   });
-
-  // Close any notifications that pop up on load. These make it impossible to
-  // interact with UI elements hidden behind them.
-  const clearableNotifications = await page.getByLabel(/Clear Notification/).all();
-  for (const notification of clearableNotifications) {
-    await notification.click();
-  }
```

Comment:
These don't pop up anymore, so leaving this block in would've failed all tests calling `openConfluentExtension` due to timing out waiting for clearable notifications.